src/options.h

```diff
@@ -52,4 +52,9 @@ class Options {
   bool perform_geometric_verification_ = false;
 };
 
+// NOT_BUILT - there is no any models
+// BUILT - models built and saved to filesystem
+// LOADED_INTO_MEMORY - models red from disk and store into Storage
+enum ReconstructionStatus { NOT_BUILT, BUILT, LOADED_INTO_MEMORY };
```

Comment:
points in the end of the comment.

why not specifying appropriate int value to every enum member?
that allows to avoid issues with input/output if we add any new States to the enum.